_posts/2017-10-04-Good-Bad-And-Ugly.md

```diff
@@ -14,49 +14,49 @@ image: 'assets/images/jsynowiec.jpg'
+One day, after stashing some changes and trying to unstash them on a different branch, unstashing failed. Up to this day I do not know why did it fail. It said that I have unresoled conflicts in files, and then gave me an empty list of files that I should resolve confilts in. I was annoyed not only because of it not working properly, but because of me losing some work that I have been doing over some time.
 
 ## Summary
 
 
 After reading this post you may think that I absolutely hate ServiceNow platform. That is completly false. I was presented to this platform as something that solves a lot of problems, a great tool that helps IT companies manage the process of producing software, platform that is easily extensible and able to adapt to requirements. Being a very sceptic person I knew there is a catch, nothing is perfect. That is why I wanted to share with everyone out there that is new to this platform as a developer some problems that I have encounter during the beginning of my journey with ServiceNow. I know I did not talk much about the good things, but I think there is enough people saying nice things about it that you could just read their posts.
 
-To give ServiceNow credit I see that there are many changes going in the right direction. Over my relativly short time spent with the platform I have seen an improvement in learning plans that are supposed to help you familiarize with how things work in ServiceNow. I have seen ServiceNow developers implement features that community requires. I also think that ServiceNow would benefit from allowing people to write their own plugins for ServiceNow that would add functionality they need. There is huge power in community collaboration over certain things. On that note, I started writing my own toolkit that would fix some of my problems listed in this article. Stay tuned for my next post, because when I finish working on the toolkit I will certanly release it for the community.
\ No newline at end of file
+To give ServiceNow credit I see that there are many changes going in the right direction. Over my relativly short time spent with the platform I have seen an improvement in learning plans that are supposed to help you familiarize with how things work in ServiceNow. I have seen ServiceNow developers implement features that the community requires. I also think that ServiceNow would benefit from allowing people to write their own plugins for ServiceNow that would add functionality they need. There is huge power in community collaboration over certain things. On that note, I started writing my own toolkit that would fix some of my problems listed in this article. Stay tuned for my next post, because when I finish working on the toolkit I will certanly release it for the community.
```

Comment:
To give ServiceNow credit I see that there are many changes going in the right direction. Over my relatively short time spent with the platform, I have seen an improvement in the learning plans that are supposed to help familiarize you with how things work in ServiceNow. I have also seen ServiceNow developers implement features that the community requires. I think that ServiceNow would benefit from allowing people to write their own plugins that could add the functionality they need. There is huge power in community collaboration on certain things. On that note, I started writing my own toolkit that would fix some of my problems listed in this post. Stay tuned for my next post, because when I finish working on the toolkit I will certainly release it for the community.